### PR TITLE
Create consignment api

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: docker/aws/deploy_lambda.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import sys
+from sessions import get_session
+
+# Create IAM client
```
Comment (suzannehamilton) [inline docker/aws/deploy_lambda.py]: Is this comment still relevant, now that the session code is in a separate file? Maybe I'm misunderstanding what you mean by "IAM client".

Comment (MancunianSam) [reply docker/aws/deploy_lambda.py]: No, it's no longer relevant, I've taken it out.

Comment (suzannehamilton) [reply docker/aws/deploy_lambda.py]: Thanks. :+1:

Same goes for update_service.py and invoke_lambda.py. Sorry, I forgot to mention that in my first comment!

Comment (MancunianSam) [reply docker/aws/deploy_lambda.py]: Ah, I should have checked those. I've removed the comments from them as well.

**Pull request merged**